content/contents+en.lr

```diff
@@ -0,0 +1,141 @@
+_model: start
+---
+title: Toolbox Bodensee e.V.
+---
+slogan: Non-profit hackspace at Lake Constance.
+---
+about_header: clubhouse
+---
+about_text: Our clubhouse is located at Bergheimer Straße 6 in Markdorf at Lake Constance in the building of intimus International GmbH.
```

Comment:
Our clubhouse is located at Bergheimer Straße 6 in Markdorf at Lake Constance on the ground of intimus International GmbH.